EdgeModuleMessageSample/ReadMe.md

```diff
@@ -0,0 +1,51 @@
+﻿# Prerequisites for EdgeModuleMessageSample
+
+Following instructions walk through steps to set-up Azure Edge device and module.
+
+## 1. Setup and deploy custom Azure IoT Edge modules:
+
+    1. Follow the article: 
```

Comment:
```suggestion
    1. Follow the steps in the article to set up Visual Studio 2022 to develop and debug modules for Azure IoT Edge
```

I think a short statement about the article is helpful here. That way, if someone has already completed this step then they don't need to click on the link, read through the entire doc and then figure out that they don't need to carry out those steps.